modules/geotiff/src/loaders.ts

```diff
@@ -0,0 +1,68 @@
+import { fromArrayBuffer } from "geotiff";
```

Comment:
Nit: following pattern in other loader modules I would name this file `geotiff-loader.ts`